### DISCUSSION TITLE
preventing auto-complete

### DISCUSSION THREAD

**Discussion by melisa-akan** [jspsych/jsPsych#1063] [Q&A] [answered]
Hello! 
 
I apologize in advance for my ignorant question, which probably has nothing to do with jsPsych. I am using the survey-text plugin to have participants type in the name of the face they see on the screen. Unfortunately, the names that I entered previously show up as I start to enter a new name (see below). This is a problem because I need people to rely on their memory. Is there any way to prevent the old responses from showing up? Again, I am sorry (and please let me know) if this question is inappropriate for this forum. 

Thank you!

<img width="262" alt="ExFace" src="https://user-images.githubusercontent.com/70907425/93542762-71fe1200-f91f-11ea-84fc-3ebad82abd8b.png">

**Comment by wenbo-lin** [comment, marked as answer]
You could add autocomplete="off" to the `<form>` tag in the survey-text plugin.

In the jspsych-survey-text.js file, you should see this line: `html += '<form id="jspsych-survey-text-form">'`

You just need to add autocomplete="off". So it's like this: `html += '<form id="jspsych-survey-text-form" autocomplete="off">'`

**Reply by melisa-akan** [reply]
It worked! Thank you so so so much @Shukichi!

**Reply by jodeleeuw** [reply]
@becky-gilbert we should probably add this as a default to the plugin, with a parameter to turn autocomplete back on?

**Reply by becky-gilbert** [reply]
Yes, definitely. I'll create an issue and mark it for the 6.2 release.
